src/helpers/useErrorHandling.ts

```diff
@@ -44,20 +44,16 @@ const useErrorHandling = (state: State) => {
     return '';
   }
 
-  const validateGradertUttak = (): { aar: string, maaneder: string } => {
+  const validateGradertUttak = (): string/* { aar: string, maaneder: string } */ => {
     let aarError = '';
```

Comment:
Samme her, nå at vi ikke behøver å skille mellom aar og maaned lenger, bør navngivning ikke la spor på legacy kode. Denne funksjonen trenger forøvrig ikke å lagre verdien i en variabel. Den kan returnere verdien direkte linje 52.